src/js/tech/ChromecastTech.js

```diff
@@ -364,6 +366,30 @@ ChromecastTech = {
       return undefined;
    },
 
+   /**
+    * This function is "required" when implementing {@link external:Tech} and is supposed
+    * to return a mock
+    * {@link https://developer.mozilla.org/en-US/docs/Web/API/TimeRanges|TimeRanges}
+    * object that represents the portions of the current media item that has playable
+    * content. However, the Chromecast API does not currently provide a way to determine
+    * how much the media item has playable content, so we'll just assume the entire video
+    * is an available seek target.
+    *
+    * The risk here lies in DVR, where ther may exist a sliding window and seeking is only
+    * possible within the last X number of minutes rather than for the entire video.
+    *
+    * Unfortunately we have no way of detecting when this is the case. Returning anything
+    * other than the full range of the video means that we lose the ability to seek during
+    * VOD. Perhaps if there's a way to detect if the source is live then we could adjust
+    * the seekable behavior accordingly.
+    *
+    * @returns {undefined} always returns `undefined`
```

Comment:
This function is returning a `TimeRanges` instance. Could you update this `@returns` statement to reflect that?

Also, could you note in the description of the `@returns` statement that the `TimeRanges` object that's returned always contains only one `TimeRange` object, and that `TimeRange` is always `0` through the `duration` of the current media item? Something like:

```
    * @returns {TimeRanges} always returns a `TimeRanges` object with one `TimeRange` that
    * starts at `0` and ends at the `duration` of the current media item
```